src/types/pathways-model.d.ts

```diff
@@ -39,12 +39,6 @@ declare module 'pathways-model' {
     action: Action[];
   }
 
-  export interface BranchState extends State {
-    criteriaSource?: string;
-    mcodeCriteria?: string;
-    otherCriteria?: string;
-  }
-
```

Comment:
Since we removed the BranchState interface we should add back the note we had before about it. 

> // NOTE: the model also includes a BranchState (which extends State),
  // but as of right now it has no additional fields not in State,
  // and TypeScript does not allow "empty" interfaces so we can't add it yet.
  // Add it here if/when we ever need it.